### ISSUE TITLE
Hyperparameters to test

### ISSUE THREAD

**Issue by EyalMK** [Adir111/TCPMSiameseBERT#12]
Test on BiLSTM hidden units - 1000
Test again on learning rate - less than 1e-5
